utils/hct/hcttest.cmd

```diff
@@ -313,7 +313,7 @@ if "%TEST_USE_LIT%"=="1" (
     set RES_DXILCONV=!RES_CLANG!
     rem check exec.
     if defined EXEC_ADAPTER (
-        py %HLSL_SRC_DIR%/utils/lit/lit.py -v --no-progress-bar --param build_mode=%BUILD_CONFIG% --param clang_site_config=%HLSL_BLD_DIR%/tools/clang/test/lit.site.cfg --param clang_taef_exec_site_config=%HLSL_BLD_DIR%/tools/clang/test/taef_exec/lit.site.cfg %EXEC_ADAPTER% %HLSL_SRC_DIR%/tools/clang/test/taef_exec
+        echo -adapter is not support when running all tests.
```

Comment:
Please change to:

The -adapter parameter is supported only when running just execution tests (hcttest.cmd exec).

Reply:
Updated.